_component_design/popovers.md

```diff
@@ -14,81 +14,183 @@ last-modified: 2017-08-17
-the UI.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The most defining aspects of a popover are the container and the pointer. The container  appears on top of the canvas. The pointer points to the popover trigger. Pointers may be used on either side of the four corners.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+- Popovers **should only be shown** when a trigger is clicked.
+- Most popovers may also have a heading and submit/cancel buttons. If buttons are not needed, a popover should close when the user clicks outside of the popover.
+- The content area may include text, inputs, or interactive content like a table or visualization.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Triggered popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered.svg){:width="100%"}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover consists of a panel, contextual pointer, heading and buttons."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-default-layout.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Triggered popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+## Pattern Specifications 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Popover Spacing
+{: .hxSubSectionTitle}
+For specs on laying out form fields and body text within a popover, refer to the following patterns:
+
+- <a href="{{site.baseurl}}/components/checkboxes.html">Checkboxes</a>
+- <a href="{{site.baseurl}}/components/dropdowns.html">Dropdowns</a>
+- <a href="{{site.baseurl}}/components/radio-buttons.html">Radio Buttons</a>
+- <a href="{{site.baseurl}}/components/type-inputs.html" target="_blank">Text Inputs</a>
+- <a href="{{site.baseurl}}/style/typography.html">Typography</a>
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Popover spacing specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-spacing.svg){:width="80%"}
 {% endfigure %}
 {% endcolumn %}
+
 </div>
 
-## Closable Popover
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Popover Specs
+{: .hxSubSectionTitle}
+For specs on laying out form fields and body text within a popover, refer to the following patterns:
+
+- Popovers **always** have a `box-shadow` attached.
+- Popover **should not** be larger than 500 x 500px. 
+
+<!-- {% include note.html content="If more space than this is required, look into -->
```

Comment:
For these comments, if they're not supposed to be visible on the page, I'd just delete them. If we want to have them available to designers, we can set them up as callouts (the `note.html` template was for the way the site looked before; we can change that, too).

Reply:
If they were from another page, I probably just commented them out as an in-the-meanwhile fix.

Reply:
Updated to```{% include note.html content="If more space than the above guide lines is required, look into solving the design challenge with a different Helix pattern."%}```